t/rose-task-run/31-app-bunch/app/bunch/rose-app.conf

```diff
@@ -0,0 +1,11 @@
+mode=rose_bunch
+meta=rose_bunch
+
+[bunch-args]
+arg1=1 2 3 4
+
+[bunch-settings]
+command-format=echo arg1: %(arg1)s
+limit=1
+instances = 4
```

Comment:
Do we want to have a bigger population? If only to test what happens when we have more than 1 digit.